pkg/network/proxy/hybridproxier.go

```diff
@@ -8,11 +8,11 @@ import (
 	"k8s.io/klog/v2"
 
 	corev1 "k8s.io/api/core/v1"
+	discoveryv1beta1 "k8s.io/api/discovery/v1beta1"
```

Comment:
you should have the v1 , it went GA in 1.21 https://github.com/kubernetes/kubernetes/pull/99870

Reply:
ah, copied this from Clayton's earlier PR...

Reply:
actually, `proxy.Provider` still uses the `v1beta1` API